### ISSUE TITLE
Status filter does not work

### ISSUE THREAD

**Issue by kaklik** [MLAB-project/MLABweb#49]
Modules in multiple statuses appear [in search results](https://www.mlab.cz/module?search=BAT&status=2) in case only one option in the filter is selected. 

![image](https://github.com/MLAB-project/MLABweb/assets/1194177/6f38c292-3f8e-4e17-83b0-b8b32003c35a)

**Comment by roman-dvorak** [comment]
Ano, při vyhledávání se neaplikují filtry. Je to schválně, aby se našly všechny moduly odpovídající vyhledávání. 

Minimálně do doby, kdy nebudou všechny aktivní moduly zelené to není dobré měnit.

**Comment by kaklik** [comment]
> Ano, při vyhledávání se neaplikují filtry. Je to schválně, aby se našly všechny moduly odpovídající vyhledávání.
> 
> Minimálně do doby, kdy nebudou všechny aktivní moduly zelené to není dobré měnit.

Myslím, že tohle by se už klidně mohlo znovu zapnout.

**Comment by kaklik** [comment]
Myslím že tohle issue je už neplatné a filtr funguje.
